extra/man/alacritty.5.scd

```diff
@@ -168,59 +179,52 @@ This section documents the *[scrolling]* table of the configuration file.
 Specifying _0_ will disable scrolling.++
 Limited to _100000_.
 
-	Default: _10000_
-
 *multiplier* <integer>
 
 	Number of line scrolled for every input scroll increment.
 
-	Default: _3_
+Default:
+
+	*[scrolling]*++
+history    = _10000_++
+multiplier = _3_
 
 # Font
 
 This section documents the *[font]* table of the configuration file.
 
 *normal* { family = <string>, style = <string> }
 
-	Default:
-		Linux/BSD: _{ family = "monospace", style = "Regular" }_++
-Windows:   _{ family = "Consolas",  style = "Regular" }_++
-macOS:     _{ family = "Menlo",     style = "Regular" }_
+	If the _style_ is not specified, it will fallback to the _default weight
+	and slant_ for this style.
 
 *bold* { family = <string>, style = <string> }
 
-	If the family is not specified, it will fall back to the value specified for
-	the normal font.
-
-	Default: _{ style = "Bold" }_
+	If the _family_ is not specified, it will fall back to the value specified
+	for the _normal font_. If the _style_ is not specified, it will fallback
+	to the _default weight and slant_ for this style.
```

Comment:
Might be nice to add a `Defaut:` "heading" here so people know that this is only relevant if they're pondering defaults?

Reply:
I don't quite understand, they could clearly rely on this behavior when omitting a specific field.

Reply:
It's a default though, so if you change it to *anything* the entire paragraph is useless.

But this is not explicitly pointed out ahead of time, basically the entire paragraph needs to be read to understand that.